### DISCUSSION TITLE
Purging Messages

### DISCUSSION THREAD

**Discussion by rekhamkodukulla** [provectus/kafka-ui#3108] [Q&A] [answered]
It would be nice to have a button to select all the topics at once , to purge the messages, rather than selecting each one

**Comment by Haarolean** [comment, marked as answer]
Hey, I guess it's already possible.

<img width="681" alt="image" src="https://user-images.githubusercontent.com/1494347/209534746-d44792c8-4670-458e-bc07-f416a5856008.png">
